stdlib/websub/src/main/ballerina/src/websub/subscriber_client.bal

```diff
@@ -36,11 +36,14 @@ public type SubscriptionClient client object {
         self.followRedirects = config?.followRedirects;
     }
 
-    # Sends a subscription request to a WebSub Hub.
-    #
-    # + subscriptionRequest - The `SubscriptionChangeRequest` containing subscription details
-    # + return - `SubscriptionChangeResponse` indicating subscription details, if the request was successful else
-    #            `error` if an error occurred with the subscription request
+# Sends a subscription request to a WebSub Hub.
+# ```ballerina
+# websub:SubscriptionChangeResponse|error response = websubHubClientEP->subscribe(subscriptionRequest);
+# ```
+#
+# + subscriptionRequest - The `SubscriptionChangeRequest` containing subscription details
+# + return - The `SubscriptionChangeResponse` indicating subscription details if the request was successful
+#           or else an `error` if an error occurred with the subscription request
```

Comment:
```suggestion
#           or else an `error` if an error occurred with the subscription request
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.